projects/documentation/content/_includes/partials/accessibility.njk

```diff
@@ -0,0 +1,84 @@
+ <div class="headerContainer">
+    <h2 id="accessibility" tabindex="-1" class="spectrum-Heading spectrum-Heading--sizeM">Accessibility testing status&nbsp;<sp-link class="header-anchor" href="#accessibility" aria-label="Accessibility" dir="ltr" focusable="">#</sp-link></h2>      
+    <sp-divider size="l" dir="ltr" role="separator"></sp-divider>
+</div>
+    <sp-table quiet style="--mod-table-row-background-color-hover: none">
+    <sp-table-head>
+        <sp-table-head-cell>Component</sp-table-head-cell>
+        <sp-table-head-cell>Accessibility test</sp-table-head-cell>
+        <sp-table-head-cell>Accessibility Contract</sp-table-head-cell>
+        <sp-table-head-cell>Status</sp-table-head-cell>
+    </sp-table-head>
+    <sp-table-body>
+        <sp-table-row value="row1">
+            <sp-table-cell><sp-link href="https://opensource.adobe.com/spectrum-web-components/components/{{componentName}}/#usage">{{componentName}}</sp-link></sp-table-cell>
+            <sp-table-cell>
+               <sp-button static="white" style="--mod-bold-font-weight:400">Default state</sp-button>
+            </sp-table-cell>
```

Comment:
We shouldn't use a button for static cell content. Also, `static="white"` is meant to render a button over a dark colored background, so that the border and focus ring will be white. Using it in this case makes the border and focus ring render as white on white, which makes it impossible to track focus.

```suggestion
            <sp-table-cell>
               Default state
            </sp-table-cell>
```